src/Components/ComNavMenu.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
use webpack plugin to avoid this annoying import in each file
```
new webpack.ProvidePlugin({
    React: 'react'
  })
```
Do not pay attention for a while on eslint warning